packages/middleware/src/handlers/prepareApiFunction/index.ts

```diff
@@ -16,6 +15,15 @@ export function prepareApiFunction(
       return;
     }
 
+    res.locals.alokai.metadata = {
+      ...res.locals.alokai.metadata,
```

Comment:
can it be possibly empty? assuming that we even added a warning for that previously it can, somehow... can we add .? safeguards with {} fallback?